<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>Имя библиотеки</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>md5 подпись</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_dll">
    <property name="title">
     <string>Библиотека</string>
    </property>
    <addaction name="action_add_dll"/>
    <addaction name="action_dll_subscribe"/>
    <addaction name="action_dll_delete"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_dll"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_exit"/>
   <addaction name="separator"/>
   <addaction name="action_add_dll"/>
   <addaction name="action_dll_subscribe"/>
   <addaction name="action_dll_delete"/>
  </widget>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="../images/resources.qrc">
     <normaloff>:/PICS/exit.png</normaloff>:/PICS/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_add_dll">
   <property name="icon">
    <iconset resource="../images/resources.qrc">
     <normaloff>:/PICS/dll_add.png</normaloff>:/PICS/dll_add.png</iconset>
   </property>
   <property name="text">
    <string>Добавить из локальной папки</string>
   </property>
  </action>
  <action name="action_dll_subscribe">
   <property name="icon">
    <iconset resource="../images/resources.qrc">
     <normaloff>:/PICS/dll_sign.png</normaloff>:/PICS/dll_sign.png</iconset>
   </property>
   <property name="text">
    <string>Подписать библиотеку</string>
   </property>
  </action>
  <action name="action_dll_delete">
   <property name="icon">
    <iconset resource="../images/resources.qrc">
     <normaloff>:/PICS/trashcan_full.png</normaloff>:/PICS/trashcan_full.png</iconset>
   </property>
   <property name="text">
    <string>Удалить запись</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../images/resources.qrc"/>
 </resources>
 <connections/>
</ui>
